specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+        }
+      }
+    },
+    "GitHubOwnerListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GitHubOwner"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubOwnerProperties": {
+      "description": "GitHub Repo Owner properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "ownerUrl": {
+          "description": "Gets or sets gitHub owner url.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubRepo": {
+      "description": "GitHub repo Proxy Resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GitHubRepoProperties"
+        }
+      }
+    },
+    "GitHubRepoListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GitHubRepo"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubRepoProperties": {
+      "description": "GitHub Repo properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "accountId": {
+          "format": "int64",
+          "description": "Gets or sets gitHub repo account id.",
+          "type": "integer"
+        },
+        "repoUrl": {
+          "description": "Gets or sets gitHub repo url.",
+          "type": "string"
+        },
+        "ownerName": {
+          "description": "Gets or sets GitHub Owner Name.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubReposProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "accountId": {
+          "format": "int64",
+          "description": "Gets or sets gitHub repo account id.",
+          "type": "integer"
+        },
+        "repoName": {
+          "description": "Gets or sets gitHub repo name.",
+          "type": "string"
+        },
+        "repoUrl": {
+          "description": "Gets or sets gitHub repo url.",
+          "type": "string"
+        }
```

Comment:
You're in the context of a GitHubRepo already, so the names 'repoName' and 'repoUrl' are partially repetitive.

'properties.repoName' might be helpful for clarity versus just 'name' which is the resource name. But I can't really say the same for 'repoUrl'. Why not just call it 'url'?

Reply:
yes, we need the disambiguation.